robot/impl/local_robot.go

```diff
@@ -548,10 +549,11 @@ func newWithResources(
 func New(
 	ctx context.Context,
 	cfg *config.Config,
+	conn rpc.ClientConn,
 	logger logging.Logger,
 	opts ...Option,
 ) (robot.LocalRobot, error) {
```

Comment:
these are breaking changes that we will have to communicate - let me know if you need help with that

Reply:
thanks!